l10n_es_aeat/models/l10n_es_aeat_map_tax_line.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2018 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+
+from odoo import fields, models
+
+
+class L10nEsAeatMapTaxLine(models.Model):
+    _name = "l10n.es.aeat.map.tax.line"
+    _order = "field_number asc, id asc"
+    _description = "AEAT tax mapping line"
+
+    field_number = fields.Integer(string="Field number", required=True)
+    tax_ids = fields.Many2many(
+        comodel_name="account.tax.template", string="Taxes templates"
+    )
+    name = fields.Char(string="Name", required=True)
+    map_parent_id = fields.Many2one(comodel_name="l10n.es.aeat.map.tax", required=True)
+    move_type = fields.Selection(
+        selection=[("all", "All"), ("regular", "Regular"), ("refund", "Refund")],
+        string="Operation type",
+        default="all",
+        required=True,
+    )
+    field_type = fields.Selection(
+        selection=[("base", "Base"), ("amount", "Amount"), ("both", "Both")],
+        string="Field type",
+        default="amount",
+        required=True,
+    )
+    sum_type = fields.Selection(
+        selection=[
+            ("credit", "Credit"),
+            ("debit", "Debit"),
+            ("both", "Both (Credit - Debit)"),
+        ],
+        string="Summarize type",
+        default="both",
+        required=True,
+    )
+    exigible_type = fields.Selection(
+        selection=[
+            ("yes", "Only exigible amounts"),
+            ("no", "Only non-exigible amounts"),
+            ("both", "Both exigible and non-exigible amounts"),
+        ],
+        required=True,
+        string="Exigibility",
+        default="yes",
+    )
+    inverse = fields.Boolean(string="Inverse summarize sign", default=False)
+    to_regularize = fields.Boolean(string="To regularize")
```

Comment:
Se añade para poder configurar un campo string para filtrado de cuentas que admita wilcard con el simbolo '%'.

`
account_expression = fields.Char(
        string="Account Expression",
        help="Only admin full account or account+%, split whit ','"
    )

    @api.constrains("account_expression")
    def _check_account_expression(self):
        for sel in self.filtered(lambda x: x.account_expression):
            for account in re.split(",[\s*]?", sel.account_expression):
                if not re.fullmatch(r"([0-9]+?%?)", account):
                    raise ValidationError(
                        _("The expression '%s' is not correct.") % sel.account_expression
                    )
`